packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -334,7 +334,7 @@ const CertificationQuiz: React.FunctionComponent = () => {
 }
 
 const CertificationQuizSkeleton: React.FunctionComponent = () => {
-  const questions = []
+  const questions: ReactNode[] = []
```

Comment:
Fix some type errors that came up, I think it may be that we have a dependent project that has more strict rules than this project (synapse-react-client), but I didn't look into it.